data/human/free worlds side plots.txt

```diff
@@ -426,6 +426,86 @@ ship "Fury" "Doombat"
 
 
 
+mission "FW Thermal Scanner 1"
+	landing
+	name "Go visit Barmy Edward"
+	description "When you have time, report to <destination>, where Barmy Edward has another device he would like you to test."
+	source
+		near "Kraz" 1 100
+	destination "Rust"
+	to offer
+		has "event: flamethrower available"
+		random < 15
+	on offer
+		conversation
+			`A message arrives for you soon after landing the <ship> on <origin>. It's from Barmy Edward, the weapons engineer at Kraz Cybernetics. He writes: "Hello again, Captain! I've been working on something new, and for some reason no one else wants to help out with the testing. If you're interested in helping drop by <destination> and come visit our factory."`
+				accept
+
+
+mission "FW Thermal Scanner 2"
+	landing
+	name "Scanner Testing"
+	description "A test ship with flamethrowers is in the <waypoints> system. Fight and disable it, keeping <npc> safe, and then go back to <destination> to give Barmy Edward the scanner logs."
+	source "Rust"
+	to offer
```

Comment:
```suggestion
	source "Rust"
	waypoint "Ildaria"
	to offer
```

Without this, the mission description text just says "A test ship with flamethrowers is in the <waypoints> system." without specifying where that is. Nor does it provide any kind of indicator without having this specified.